test-cases/performance/perf-search-best-throughput-config.yaml

```diff
@@ -16,12 +16,30 @@ stress_step_duration: '15m'
 max_deviation: 5 # 5% relative difference
 n_stress_process: 1
 
+#AWS
 instance_type_db: 'i2.2xlarge'
 instance_type_loader: 'c4.2xlarge'
 instance_type_monitor: 't3.small'
+space_node_threshold: 644245094
+# tests are running on aws. The loader uses the shard-aware driver
+ami_id_loader: 'scylla-qa-loader-ami-v19'
+#--------
+#GCE
+ce_instance_type_db: 'n2-highmem-8'
+gce_root_disk_type_db: 'pd-ssd'
+gce_n_local_ssd_disk_db: 8
+gce_instance_type_loader: 'c3-highcpu-8'
+gce_root_disk_type_loader: 'pd-ssd'
+root_disk_size_loader: 50
+gce_instance_type_monitor: 'n2-highmem-8'
+#+ rewrite vars from  perf-regression-throughput-config-gce-modifications.yaml
+#use_preinstalled_scylla: true
+#space_node_threshold: 0
+#ami_id_loader: ''
```

Comment:
same?

Reply:
do you mean we can leave ami_id_loader: 'scylla-qa-loader-ami-v19' for GCE?

Reply:
I meant same comments as above for all the commented params.